src/Frontend/components/app/layout/Application.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * This file is part of the Sententiaregum project.
+ *
+ * (c) Maximilian Bosch <maximilian.bosch.27@gmail.com>
+ * (c) Ben Bieler <benjaminbieler2014@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+import React from 'react';
+import menu from '../../../config/menu';
+import AppMenu from './AppMenu';
+
+/**
+ * Container for the whole app.
+ *
+ * @param {Object} props The properties of the app container.
+ *
+ * @returns {React.Element} The markup of the app.
+ */
+const Application = props => {
+  return (
```

Comment:
__NOTE:__ I didn't implement a refresh menu feature which re-renders the menu structure if something changes. Instead it should listen to the login/logout lifecycle to be implemented in #35 and decide based on the given data of the login/logout cycle whether to re-render. This saves unncessary resources that would be needed to re-evaluate the menu items at every hash-change